src/main/java/com/takipi/oss/benchmarks/jmh/loops/ByteArrayLoopBenchmark.java

```diff
@@ -0,0 +1,107 @@
+package com.takipi.oss.benchmarks.jmh.loops;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+
+@State(Scope.Benchmark)
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
+@Fork(2)
+@Measurement(iterations = 5)
+@Warmup(iterations = 5)
+public class ByteArrayLoopBenchmark {
+    int size = 100_000;
+
+    int byteArrLength = 100;
+
+    List<byte[]> arrays;
+
+    @Setup
+    public void setup() {
+        arrays = new ArrayList<>(size);
+
+        Random random = new Random();
```

Comment:
Fair enough. I still think that's pretty unlikely to ever occur, but I also don't ever want to worry about it. :) After all, "pretty unlikely" isn't good enough for thread safety!